cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -291,6 +297,12 @@ func (s *Supervisor) getBootstrapInfo() (err error) {
 		return err
 	}
 
+	defer func() {
+		if stopErr := cmd.Stop(context.Background()); stopErr != nil {
+			err = errors.Join(err, fmt.Errorf("stop collector command: %w", stopErr))
```

Comment:
```suggestion
			err = errors.Join(err, fmt.Errorf("error when stopping the collector: %w", stopErr))
```
Nit

Reply:
@BinaryFissionGames What do you think about this wording? It's a nit, but I think this is a bit clearer.

Reply:
I must have dropped it when rebasing at some point, I definitely meant to take these suggestions.